services/app-api/emailer/emailer.ts

```diff
@@ -81,40 +92,49 @@ function newSESEmailer(config: EmailConfiguration): Emailer {
             )
             return await this.sendEmail(emailData)
         },
-        sendUnlockPackageCMSEmail: async function (unlockEmailData){
-            const emailData = unlockPackageCMSEmail(unlockEmailData, config)
+        sendUnlockPackageCMSEmail: async function (updatedEmailData){
+            const emailData = unlockPackageCMSEmail(updatedEmailData, config)
             return await this.sendEmail(emailData)
         },
-        sendUnlockPackageStateEmail: async function (submission, unlockEmailData){
+        sendUnlockPackageStateEmail: async function (submission, updatedEmailData){
             const emailData = unlockPackageStateEmail(
                 submission,
-                unlockEmailData, 
+                updatedEmailData,
                 config)
             return await this.sendEmail(emailData)
+        },
+        sendResubmittedStateEmail: async function (
+            submission,
+            updatedEmailData,
+            user: CognitoUserType
+        ){
+            const emailData = resubmittedStateEmail(submission, user, updatedEmailData, config)
+            return await this.sendEmail(emailData)
+        },
+        sendResubmittedCMSEmail: async function (submission, updatedEmailData){
+            const emailData = resubmittedCMSEmail(submission, updatedEmailData, config)
+            return await this.sendEmail(emailData)
         }
     }
 }
 
+const localEmailerLogger = (emailRequestParams: SES.SendEmailRequest) =>
+    console.log(`
+        EMAIL SENT
+        ${'(¯`·.¸¸.·´¯`·.¸¸.·´¯·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´)'}
+        ${JSON.stringify(emailRequestParams)}
+        ${'(¯`·.¸¸.·´¯`·.¸¸.·´¯·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´)'}
+    `)
+
+
 function newLocalEmailer(config: EmailConfiguration): Emailer {
     return {
         sendEmail: async (emailData: EmailData): Promise<void | Error> => {
-            const emailRequestParams = getSESEmailParams(emailData)
-            console.log(`
-            EMAIL SENT
-            ${'(¯`·.¸¸.·´¯`·.¸¸.·´¯·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´)'}
-            ${JSON.stringify(emailRequestParams)}
-            ${'(¯`·.¸¸.·´¯`·.¸¸.·´¯·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´)'}
-        `)
+            localEmailerLogger(getSESEmailParams(emailData))
```

Comment:
You could put `getSEESEmailParams` into `localEmailerLogger` as well since we call it in every instance here.